src/realm/cuda/cuda_internal.cc

```diff
@@ -1570,7 +1667,7 @@ namespace Realm {
         // self-path
         unsigned bw = src_gpu->info->logical_peer_bandwidth[_src_gpu->info->index];
         unsigned latency = src_gpu->info->logical_peer_latency[_src_gpu->info->index];
-        unsigned frag_overhead = 2000; // HACK - estimate at 2 us
+        unsigned frag_overhead = 200; // HACK - estimate at 2 us
```

Comment:
Why change it to 200?

Reply:
I think I'd like an answer to this as well